builder/builder.go

```diff
@@ -0,0 +1,82 @@
+package builder
+
+import (
+	"archive/tar"
+	"bytes"
+	"context"
+	"github.com/docker/docker/api/types"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/docker/docker/client"
+)
+
+// Builds Docker Image for Code Runner.
+func BuildCodeRunnerImage() error {
```

Comment:
Possibly change this to `BuildImage()` as the image will eventually not just be for the code-runner.

Reply:
Done